playbooks/common/openshift-cluster/upgrades/pre/verify_upgrade_targets.yml

```diff
@@ -27,13 +27,17 @@
 
   - name: Set fact avail_openshift_version
     set_fact:
-      avail_openshift_version: "{{ repoquery_out.results.versions.available_versions.0 }}"
+      avail_openshift_version: "{{ repoquery_out.results.versions.available_versions_full.0 }}"
+  - name: Set openshift_pkg_version when not specified
+    set_fact:
+      openshift_pkg_version: "-{{ repoquery_out.results.versions.available_versions_full.0 }}"
+    when: openshift_pkg_version | default('') == ''
 
   - name: Verify OpenShift RPMs are available for upgrade
     fail:
       msg: "OpenShift {{ avail_openshift_version }} is available, but {{ openshift_upgrade_target }} or greater is required"
     when:
-    - avail_openshift_version | default('0.0', True) | version_compare(openshift_release, '<')
+    - openshift_pkg_version | default('0.0', True) | version_compare(openshift_release, '<')
```

Comment:
We should check `openshift_pkg_version < 3.8` and fail if true

Reply:
Plus, we should move the tasks from `Check latest available OpenShift RPM version` to `Verify OpenShift RPMs are available for upgrade` to the `openshift_version` role (once the docker dep is gone).

Reply:
None of the nits are blockers.

Reply:
It'd be nice if we could do < upgrade_target + .1 here so we don't have the target version embedded all over the code base.

Reply:
Seems {{ avail_openshift_version }} should be {{ openshift_pkg_version }} here